languages/csharp/exercises/concept/nullability/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+In this exercise you'll be writing code to helping printing name
+badges for factory employees.
+
+Employees have an id, a name and a department name.  For example, if
+my id is `734`, my name is `Ernest Johnny Payne`, and my department is
+`Strategic Communication` my badge name is: `[734] - Ernest Johnny
+Payne - STRATEGIC COMMUNICATION`.
```

Comment:
I was thinking that maybe we can draw inspiration from the [strings exercise](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/strings/.docs/instructions.md) and just specify the format. That way, we can lose the example here and use those values for the sample code that is accompanied

```suggestion
Employees have an ID, name and a department. Badge labels are formatted as follows: `"[<ID>] - <NAME> - [DEPARTMENT]"`.
```

Maybe we can the first task _not_ have the uppercase requirement? That way, we can have multiple tasks and gradually introduce the null-reference problem. In fact, not having the requirement at first will make it more likely that students _will_ encounter the null-reference, so that is great :)